bin/foxlator-tools.py

```diff
@@ -1,16 +1,30 @@
+#!/usr/bin/env python3.10
+
 import argparse
+import subprocess
 from enum import Enum
+
+import foxlator_lib as fll
```

Comment:
I noticed that we should add a stub file to foxlator_lib. I think we should also add "__all__ = ["utils"]" to the end of the __init__ file

Reply:
We decided to address that under a separated ticket.